src/components/Operatives/OperativeWorkOrderDetails.js

```diff
@@ -6,112 +6,118 @@ import { GridColumn, GridRow } from '../Layout/Grid'
-        </GridColumn>
-        <GridColumn width="one-half" className="align-grid-column">
-          <div className="lbh-body">{workOrder.callerNumber}</div>
-        </GridColumn>
-      </GridRow>
-    )}
+      {workOrder.plannerComments && (
+        <>
+          <div className="lbh-heading-h5">Planner Comment</div>
+          <p className="govuk-body">{workOrder.plannerComments}</p>
+        </>
+      )}
+
+      {personAlerts.length > 0 && (
+        <GridRow>
+          <GridColumn width="one-half">
+            <div className="lbh-heading-h5">Caut. contact</div>
+          </GridColumn>
+          <GridColumn width="one-half" className="align-grid-column">
+            <div className="govuk-warning-text lbh-warning-text">
+              <span
+                class="govuk-warning-text__icon person-alert--icon"
+                aria-hidden="true"
+              >
+                !
+              </span>
+              <strong className="govuk-warning-text__text person-alert--text">
+                {personAlerts.map((alert) => alert.type).join(',')}
+              </strong>
+            </div>
+          </GridColumn>
+        </GridRow>
+      )}
 
-    {workOrder.appointment && workOrder.appointment.note && (
       <GridRow>
         <GridColumn width="one-half">
-          <div className="lbh-heading-h3">Comment</div>
+          <div className="lbh-body property-name">Address</div>
         </GridColumn>
         <GridColumn width="one-half" className="align-grid-column">
-          <div className="lbh-body">{workOrder.appointment.note}</div>
+          <div className="lbh-body">
+            {property.address.shortAddress} {property.address.postalCode}
+          </div>
         </GridColumn>
       </GridRow>
-    )}
+
+      {workOrder.callerName && (
+        <GridRow>
+          <GridColumn width="one-half">
+            <div className="lbh-body property-name">Contact</div>
+          </GridColumn>
+          <GridColumn width="one-half" className="align-grid-column">
+            <div className="lbh-body">{workOrder.callerName}</div>
+          </GridColumn>
+        </GridRow>
+      )}
+
+      {workOrder.callerNumber && (
+        <GridRow>
+          <GridColumn width="one-half">
+            <div className="lbh-body property-name">Telephone no</div>
+          </GridColumn>
+          <GridColumn width="one-half" className="align-grid-column">
+            <div className="lbh-body">{workOrder.callerNumber}</div>
+          </GridColumn>
+        </GridRow>
+      )}
+
+      {workOrder.appointment && workOrder.appointment.note && (
+        <GridRow>
+          <GridColumn width="one-half">
+            <div className="lbh-body property-name">Comment</div>
+          </GridColumn>
+          <GridColumn width="one-half" className="align-grid-column">
+            <div className="lbh-body">{workOrder.appointment.note}</div>
+          </GridColumn>
+        </GridRow>
+      )}
+    </div>
   </>
 )
 
```

Comment:
We're having `locationAlerts` and `tenure` as required for this view which we don't. I am going to make a PR for this and then merge